Src/FluentAssertions/Primitives/StringAssertions.cs

```diff
@@ -110,11 +110,40 @@ public AndConstraint<TAssertions> BeOneOf(IEnumerable<string> validValues, strin
     /// <param name="becauseArgs">
     /// Zero or more objects to format using the placeholders in <paramref name="because" />.
     /// </param>
-    public AndConstraint<TAssertions> BeEquivalentTo(string expected, string because = "",
-        params object[] becauseArgs)
+    public AndConstraint<TAssertions> BeEquivalentTo(string expected,
```

Comment:
🤔 I think we also need to add this to the `NotBeEquivalentTo` API for consistency.

Reply:
I added the additional API changes in #2364 and would implement it in a separate pull request.

Reply:
IMHO one PR should consist of both the positive and the negative (if applicable)

Reply:
Yeah, that's the idea. This also avoids the situation where somebody implements one part and then decides to abandon the the work.

Reply:
I added the overload for `NotBeEquivalentTo`.